release/create_release_archives.sh

```diff
@@ -29,9 +29,16 @@ ISTIO_VERSION="$(cat ${ROOT}/istio.RELEASE)"
 [[ -z "${ISTIO_VERSION}" ]] && error_exit 'ISTIO_VERSION is not set'
 [[ -z "${ISTIOCTL_URL}" ]] && error_exit 'ISTIOCTL_URL is not set'
 
-TMP_DIR="$(mktemp -d /tmp/istio.version.XXXX)"
-COMMON_FILES_DIR="${TMP_DIR}/istio/istio-${ISTIO_VERSION}"
-ARCHIVES_DIR="${TMP_DIR}/archives"
+BASE_DIR="$(mktemp -d /tmp/istio.version.XXXX)"
```

Comment:
Indeed. I overlooked the fact that a directory is actually created besides returning a path. Good catch.